app/js/components/LabResultsList.jsx

```diff
@@ -351,8 +403,8 @@ export class LabResultsList extends PureComponent {
       errorMessage = "",
     } = selectedPatient;
 
+    // build a list of all obs from the encounters that are of type LabSet or Test
```

Comment:
I noticed that this view operates by fetching *all* encounters that potentially have lab results and then fetching the results off them.  This seems like it could be perform pretty badly, but that's an existing issue, so will handle on another ticket.